stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -2548,6 +2810,16 @@ impl NakamotoChainState {
             );
         }
 
+        // Handle signer stackerdb updates
+        if evaluated_epoch >= StacksEpochId::Epoch25 {
+            let _events = Self::check_and_handle_prepare_phase_start(
```

Comment:
I wasn't sure if we needed to propagate these events and if so, where?

Reply:
For now, I think it's fine to not propagate any events. If we want events here for the dispatcher, we can figure that out later.